docs/host/host.md

```diff
@@ -251,3 +251,32 @@ When Ksmtuned starts, initialize `pages_to_scan` in KSM to 1000 (**Minimum Pages
 KSM starts when the available memory falls below the **Threshold Coefficient**. If it detects that it is running, `pages_to_scan` increments by 300 (**Boost**) every minute until it reaches 5000 (**Maximum Pages**).
 
 KSM will stop when the available memory is above the **Threshold Coefficient**. If it detects that it is stopped, `pages_to_scan` decrements by 100 (**Decay**) every minute until it reaches 1000 (**Minimum Pages**).
+
+## NTP Configuration
+
+Time synchronization is important for the distributed cluster architecture, so there is a simple way to configure the NTP servers on Harvester.
```

Comment:
```suggestion
Time synchronization is an important aspect of distributed cluster architecture. Because of this, Harvester now provides a simpler way for configuring NTP settings.
```

Reply:
Because the Harvester UI uses the term "Settings", we should make it easy for users to map the concepts to that particular screen. Here is the wording convention that I used:
- "configure" and "configuration": refer to the action the user performs
- "setting": refers to what the user configures